components/drops/useDrops.ts

```diff
@@ -55,41 +56,28 @@ export function useDrops(query?: GetDropsQuery) {
   const count = computed(() => dropsList.value.length)
   const loaded = ref(false)
 
-  onMounted(async () => {
+  onBeforeMount(async () => {
     dropsList.value = await getDrops(query)
 
-    Promise.all(
-      dropsList.value
-        .filter((drop) => !isProduction || drop.chain !== 'ahk')
-        .map((drop) => {
-          return new Promise((resolve) => {
-            const { result: collectionData } = useQuery(
-              unlockableCollectionById,
-              { id: drop.collection },
-              { clientId: drop.chain },
-            )
-
-            watchEffect(async () => {
-              if (collectionData.value?.collectionEntity) {
-                const { collectionEntity } = collectionData.value
-                const newDrop = await getFormattedDropItem(
-                  collectionEntity,
-                  drop,
-                )
-                resolve(newDrop)
-              }
-            })
-          })
-        }),
-    ).then((dropsDataList) => {
-      drops.value = sortBy(dropsDataList as Drop[], (drop) =>
-        DROP_LIST_ORDER.indexOf(drop.status),
-      )
-      loaded.value = true
-    })
+    dropsList.value
+      .filter((drop) => !isProduction || drop.chain !== 'ahk')
```

Comment:
Is it still necessary to have this filter as you have changed the config.

<img width="389" alt="image" src="https://github.com/kodadot/nft-gallery/assets/31397967/f8af9edc-93f9-4e61-a866-d6c0cfe9013b">